### PR TITLE
For #1649: Resume denial.

### PR DESCRIPTION
**Pull request by paulodamaso**
For #1649:
- Added tests for checking if examiner received points for user resume denial, if examiner is not responsible for user resume anymore and if examiner received message.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1655](https://github.com/zerocracy/farm/issues/1655) is now in scope, role is `REV`

File: src/test/resources/com/zerocracy/bundles/deny_resume_application/_before.groovy [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import com.zerocracy.pmo.Awards
+import com.zerocracy.pmo.Resumes
+
+import java.time.LocalDateTime
+
+def exec(Project project, XML xml) {
+  Farm farm = binding.variables.farm
+  Resumes resumes = new Resumes(farm).bootstrap()
+  resumes.add(
+    'friend',
```
Comment (krzyk) [inline src/test/resources/com/zerocracy/bundles/deny_resume_application/_before.groovy]: @paulodamaso this should be a variable, it is used in two places here

File: src/test/resources/com/zerocracy/bundles/deny_resume_application/_after.groovy [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+import com.jcabi.xml.XML
+import com.zerocracy.Project
+
+def exec(Project project, XML xml) {
+//  Farm farm = binding.variables.farm
+//  People people = new People(farm).bootstrap()
+//  String friend = 'friend'
+//  MatcherAssert.assertThat(
+//    people.hasMentor(friend),
```
Comment (krzyk) [inline src/test/resources/com/zerocracy/bundles/deny_resume_application/_after.groovy]: @paulodamaso actually it should test if that person exists, so `people.exists(friend)`.

File: src/test/resources/com/zerocracy/bundles/deny_resume_application/_after.groovy [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ * SOFTWARE.
+ */
+package com.zerocracy.bundles.deny_resume_application
+
+import com.jcabi.xml.XML
+import com.zerocracy.Project
+
+def exec(Project project, XML xml) {
+//  Farm farm = binding.variables.farm
+//  People people = new People(farm).bootstrap()
```
Comment (krzyk) [inline src/test/resources/com/zerocracy/bundles/deny_resume_application/_after.groovy]: @paulodamaso this can be inlined

**Comment by paulodamaso** [comment]
@krzyk Thanks for the comments, please take a look.

File: src/test/resources/com/zerocracy/bundles/deny_resume_application/_after.groovy (hunk 2 above)
Comment (krzyk) [reply src/test/resources/com/zerocracy/bundles/deny_resume_application/_after.groovy]: @paulodamaso I thought the user is not created until it is invited by someone (so after resume approval).

**Comment by krzyk** [comment]
@paulodamaso thanks

**Comment by krzyk** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 35 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1655](https://github.com/zerocracy/farm/issues/1655) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@krzyk According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one major issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by krzyk** [comment]
@ypshenychka I confirm

**Comment by ypshenychka** [comment]
@krzyk thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Pull request merged**